Toolset/palettes/script editor/behaviors/revseeditorbehavior.livecodescript

```diff
@@ -951,7 +951,7 @@ command scriptComment
    # OK-2009-03-03 : Bug 7537 - Reformat the handler after commenting, this may be flawed,
    # but the proper solution is rather time-consuming.
```

Comment:
Now that I change it to script scope commenting out a handler in the IDE library is sloooow... I think this needs some more logic appied